nixos/tests/portunus.nix

```diff
@@ -0,0 +1,19 @@
+import ./make-test-python.nix ({ lib, ... }:
+
+{
+  name = "portunus";
+  meta.maintainers = with lib.maintainers; [ SuperSandro2000 ];
+
+  nodes.machine = _: {
+    services.portunus = {
+      enable = true;
+      ldap.suffix = "dc=example,dc=org";
+    };
+  };
+
+  testScript = ''
+    machine.wait_for_unit("portunus.service")
+    # machine.wait_for_console_text("Editor is now accessible via")
```

Comment:
drop the comment if it doesn't work

Reply:
just forget to delete that from the copied test